packages/sync/src/androidMain/kotlin/io/realm/sync/transport/NetworkTransport.kt

```diff
@@ -0,0 +1,2 @@
+package io.realm.sync.transport
```

Comment:
In Java, this package is called `io.realm.mongodb` with a `io.realm.mongodb.sync` subpackage. IMO we should do the same here.